lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs

```diff
@@ -923,6 +949,28 @@ testRule({
 	],
 });
 
+testRule({
+	ruleName,
+	config: [true, { ignoreShorthands: ['/^font-variant/'] }],
```

Comment:
[suggest] `ignoreShorthands` has been tested already, so can you remove this case?

Ref https://github.com/stylelint/stylelint/pull/7734/files#diff-1f86a8d8167e5474d3acb984827780d6f4fdc4b66d36478c75280f22c7e2b3cbR976